src/components/StakingVaults/Cells.tsx

```diff
@@ -99,9 +101,9 @@ export const AssetCell = ({
       <HStack flex={1}>
         <SkeletonCircle isLoaded={!!asset} mr={2} width='auto' height='auto'>
           {icons && icons.length > 1 ? (
-            <PairIcons icons={icons} iconSize='sm' bg='none' />
+            <PairIcons icons={icons} iconSize='sm' bg='none' showFirst={true} />
           ) : (
-            <AssetIcon assetId={asset.assetId} size='md' />
+            <AssetIcon assetId={asset.assetId} size='md' pairProps={pairProps} />
```

Comment:
Consumed in quite a few places, test me against release there
<img width="851" alt="image" src="https://github.com/user-attachments/assets/3b6f8094-846c-4b78-b8a8-345427c02ccf">

Reply:
✅ 
While it *is* consumed in a bunch of places (fiat ramps, markets page, your loans, available pools, related assets...) you won't see LP assets there, so effectively a low surface area - gave it some smoke test in all those places and couldn't get to any LP assets even after going many, many pages on fiat ramps/markets